physics/SFC_Models/Land/Noahmp/module_sf_noahmplsm.F90

```diff
@@ -5727,6 +5755,17 @@ subroutine gfs_stability                                              &
 
           endif          ! end of if (dtv >= 0 ) then loop
 !
+! form drag coefficient
+!
+          if ( varf.gt.varf_min .and. psl_gwd_z0m_factor.gt.0.0 ) then
+            zf    = min( min(varf,varf_max)*psl_gwd_z0m_factor,z1 )
+            fri   = min( max( 1.-rb,0. ), 1.)
+            fphim    = log( ( z1 + zf) / zf )
```

Comment:
add a comment for what `fphim` represents

Reply:
changed